.env

```diff
@@ -0,0 +1,7 @@
+REACT_APP_apiKey=AIzaSyAMjsmLhc67QdCKNDvDpO1RHlTjmZwhB4A
+REACT_APP_authDomain=mindfullspace-22.firebaseapp.com
+REACT_APP_databaseURL=https://mindfullspace-22-default-rtdb.firebaseio.com
+REACT_APP_projectId=mindfullspace-22
+REACT_APP_storageBucket=mindfullspace-22.appspot.com
+REACT_APP_messagingSenderId=560746506022
+REACT_APP_appId=1:560746506022:web:526eb92f95a17af3eec3e7
```

Comment:
.env files should not be on the repo, but in the service you use to deploy the site